packages/components/CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 ## Unreleased
 
+### New Feature
+
+-  `ColorPalette`, `GradientPicker`, `PaletteEdit`, `ToolsPanel`: add new props to set a custom heading level ([43848](https://github.com/WordPress/gutenberg/pull/43848) and [#47788](https://github.com/WordPress/gutenberg/pull/47788)).
+
```

Comment:
```suggestion
### Enhancements

-  `ColorPalette`, `GradientPicker`, `PaletteEdit`, `ToolsPanel`: add new props to set a custom heading level ([#43848](https://github.com/WordPress/gutenberg/pull/43848) and [#47788](https://github.com/WordPress/gutenberg/pull/47788)).

```

Nit: What do you think about consolidating the "New Feature" headings into "Enhancements"? This has been a tiny pet peeve of mine because they are kind of the same thing, especially in terms of semver.

Reply:
Sure thing, sounds good to me!